config/default.schema.json

```diff
@@ -1675,6 +1675,16 @@
 			"type": "boolean",
 			"default": "false",
 			"description": "Enables the external system logout feature"
+		},
+		"VIDIS_API_CLIENT_BASE_URL": {
+			"type": "string",
+			"default": "https://service-stage.vidis.schule/o/vidis-rest",
```

Comment:
The default.schema.json should only contain default values that would also be valid in production for all instances. Defining a default also prevents it from being undefined, but the code assumes that it could be undefined. (Same goes for VIDIS_SYNC_REGION)